config/routes.rb

```diff
@@ -1,13 +1,15 @@
 Rails.application.routes.draw do
 
+  
+
    devise_for :users
 
 
    resources :questions
    resources :users
    resources :topics do
-     resources :posts, except: [:index]do
-       resources :summary, except: [:edit, :update]
+     resources :posts do
+       resources :summaries
```

Comment:
Can a post have one `summary` or multiple?

Since it can only have on the routes are different.  IE it wouldn't make since to look at `posts/1/summary/2` when you could just do `posts/1/summary`.  Therefore, you can use [singular resources](http://guides.rubyonrails.org/routing.html#singular-resources) as indicated by the link in the assignment.